data/wanderer/wanderers start.txt

```diff
@@ -2612,6 +2612,7 @@ mission "Wanderers Ap'arak 3"
 		has "Wanderers Ap'arak 2: done"
 	on offer
 		log `The Wanderers have begun producing even larger warships, and have succeeded in driving off yet another Unfettered assault. Afterwards the Pug, whom the Wanderers refer to as the "Keepers," finally sent assistance... in the form of two ships, which they claim are powerful enough to keep the entire Unfettered fleet at bay.`
+		log "Factions" "Unfettered Hai" `After obtaining a sufficient number of Jump Drives, the Unfettered expanded into Wanderer space and managed to conquer 3 worlds from the Wanderers until their advance was stopped by the interference of the Pug.`
```

Comment:
```suggestion
		log "Factions" "Unfettered Hai" `After obtaining a sufficient number of Jump Drives, the Unfettered expanded into Wanderer space and managed to conquer 3 worlds from the Wanderers before their advance was stopped due to Pug interference.`
``` 
until --> before | I think it sounds better
Also shortened up the very end of the sentence.